inc/suit_common.h

```diff
@@ -12,24 +12,24 @@
 #include <stdlib.h>
 #include "qcbor/qcbor.h"
 
-
 #define SUIT_FAILED_TO_VERIFY               6 // COSE verification failure or hash digest failure
 #define SUIT_NOT_IMPLEMENTED                5 // parser is not implemented
 #define SUIT_NO_MORE_ITEMS                  4 // mandatory items in array is not appeared
 #define SUIT_INVALID_TYPE_OF_ARGUMENT       3 // type of an item is not expected
 #define SUIT_NO_MEMORY                      2 // items exceed the allocated memory
 #define SUIT_FATAL_ERROR                    1 // unknown error
-#define SUIT_SUCCESS                        0
+#define SUIT_SUCCESS                        0 // success
 
-#define SUIT_DECODE_MODE_STRICT                 0b00000000 // abort immediately on any error
-#define SUIT_DECODE_MODE_SKIP_SIGN_FAILURE      0b00000001 // through but report on verification failure
-#define SUIT_DECODE_MODE_SKIP_UNKNOWN_ELEMENT   0b00000010 // through unknown or unimplemented element(key or value)
-#define SUIT_DECODE_MODE_PRESERVE_ON_ERROR      0b00000100 // preserve successfully parsed elements on error in Map/Array
-#define SUIT_DECODE_MODE_SKIP_ANY_ERROR         0b11111111 // through excepting fatal error
+#define SUIT_DECODE_MODE_STRICT                 0 // 0b00000000 - abort immediately on any error
```

Comment:
I made this change because this works with GNU C compilers but not with others

Reply:
I forgot that! Thank you.